tests/code_size/random_printf_wasm2js.json

```diff
@@ -0,0 +1,6 @@
+{
+  "a.html": 20613, 
+  "a.html.gz": 8602, 
+  "total": 20613, 
+  "total_gz": 8602
+}
```

Comment:
Add newlines to EOF.

Reply:
These are programmatically generated. Could add an explicit `+ '\n'` to the write, but why? It is not as if these files are #included anywhere that we'd need to care about concatenation rules...

Reply:
Yeah, please just add the '\n' when you write the file.

For consistency I think all text files (especially those under source control) should end in a newline.   Tools like vim and github's UI hightlight the lack of newlines and I tend to agree with them.